src/lib/Components/Consignments/Screens/Metadata.tsx

```diff
@@ -0,0 +1,154 @@
+const categoryOptions = [
+  "Painting",
+  "Sculpture",
+  "Photography",
+  "Print",
+  "Drawing, Collage or other Work on Paper",
+  "Mixed Media",
+  "Performance Art",
+  "Installation",
+  "Video/Film/Animation",
+  "Architecture",
+  "Fashion Design and Wearable Art",
+  "Jewelry",
+  "Design/Decorative Art",
+  "Textile Arts",
+  "Other",
+]
+
+interface State extends ConsignmentMetadata {
+  showSelector?: boolean
+}
+
+export default class Metadata extends React.Component<Props, State> {
+  constructor(props) {
+    super(props)
+    this.state = props
+  }
+
+  doneTapped = () => {
+    this.props.updateWithMetadata(this.state)
+    this.props.navigator.pop()
+  }
+
+  updateSigned = () => this.setState({ unit: this.state.unit === "cm" ? "in" : "cm" })
+  updateTitle = title => this.setState({ title })
+  updateYear = year => this.setState({ year })
+  updateCategory = category => this.setState({ category })
+  updateMaterials = materials => this.setState({ materials })
+  updateWidth = width => this.setState({ width })
+  updateHeight = height => this.setState({ height })
+  updateDepth = depth => this.setState({ depth })
+
+  animateStateChange = newState => {
+    const animate = LayoutAnimation.easeInEaseOut as any
+    animate()
+    this.setState(newState)
+  }
+
+  showCategorySelection = () => this.animateStateChange({ showSelector: true })
+  hideCategorySelection = () => this.animateStateChange({ showSelector: false })
+
+  render() {
+    return (
+      <ConsignmentBG>
+        <DoneButton onPress={this.doneTapped}>
+          <ScrollView>
+            <View>
+              <Row>
+                <Text
+                  text={{ placeholder: "Title", onChangeText: this.updateTitle, value: this.state.title }}
+                  style={{ margin: 10 }}
+                />
+              </Row>
+
+              <Row>
+                <Text
+                  text={{ placeholder: "Year", onChangeText: this.updateYear, value: this.state.year }}
+                  style={{ margin: 10 }}
+                />
+              </Row>
+
+              <Row>
+                <Text
+                  text={{
+                    placeholder: "Category",
+                    onChangeText: this.updateCategory,
+                    onFocus: this.showCategorySelection,
+                    value: this.state.category,
+                  }}
+                  style={{ margin: 10 }}
+                />
+              </Row>
+
+              {this.state.showSelector
+                ? <Picker
+                    style={{ backgroundColor: "white" }}
+                    selectedValue={this.state.category}
+                    onValueChange={(itemValue, itemIndex) => this.setState({ category: itemValue })}
+                  >
+                    {categoryOptions.map(category => <Picker.Item label={category} value={category} />)}
+                  </Picker>
+                : null}
```

Comment:
There isn't a popover in iOS, which the designs imply, to have _something_ I've added a spinny picker.